src/modules/cookie-preference/index.js

```diff
@@ -0,0 +1,60 @@
+import jsCookies from '../../../node_modules/js-cookie/src/js.cookie';
+
+function createCookiePreference(cookieName, allowedPreferences) {
+    const cookieConfig = {
+        path: '/',
+        expires: 365,
+        samesite: 'strict'
+    };
+
+    function getPreference(preferenceName) {
+        let cookieValue = jsCookies.get(cookieName) || '{}';
```

Comment:
The js-cookie module has a [native JSON api](https://www.npmjs.com/package/js-cookie#json)

Can we use it and save handling it in this wrapper? e.g.

```js
Cookies.set('name', { foo: 'bar' });
Cookies.getJSON('name'); // => { foo: 'bar' }
```